VideoLocker/src/main/java/org/edx/mobile/base/MainApplication.java

```diff
@@ -150,6 +158,31 @@ public void onApplicationLaunchedFromBackground(){
     }
 
 
+    /**
+     * if app is launched from upgrading, we need to resync with parse server.
+     * @param context
+     */
+    private void tryToUpdateParseForAppUpgrade(Context context){
+
+        PrefManager.AppInfoPrefManager pmanager = new PrefManager.AppInfoPrefManager(context);
+        Long preVersion = pmanager.getAppVersionCode();
```

Comment:
let's just call this ``previousVersion``? The abbreviation doesn't seem worth the loss of clarity